docs/getting-started-using-a-plan.md

```diff
@@ -2,108 +2,145 @@
 
-Plugin Subpanel
+On the left of this panel is the **plugin panel**.
+
+Plugin Panel
 ---------------
 
-The Plugin Subpanel displays a list of Molecule plugins, that adapts to the
-current project by only displaying plugins for tools the project actually uses.
+The plugin sub-panel displays a list of Molecule plugins available for tools
+the project actually uses. Plugins related to technologies not used in the project
+are hidden.
 
 Molecule decides which tools the project uses by browsing for files that we call
 **Packages**. For instance, if the project directory has an `.eslintrc` file,
 then Molecule considers that the project uses the Eslint tool, and displays the
-Eslint plugin in the Plugin Subpanel.
+Eslint plugin in the Plugin Panel.
 
 Which files count as a package varies from plugin to plugin. Most Node.js
 plugins consider `package.json` to be a package file if the corresponging tool
 is installed in `node_modules`.
 
 ### Plans
 
-To execute these plugins, you need to create a **Molecule plan**. A plan is a
-reusable action that executes a given plugin with pre-registered parameters. For
-instance, you can create a plan for the NPM plugin that will always call
-`npm run start`.
+To execute these plugins, you need to create a **plan**. A plan is a
+configuration that executes a given plugin with pre-registered parameters. For
+instance, you can create a plan for NPM that will always call `npm run start`.
+
+Each plugin is displayed with a list of  pre-generated plans and an option to
+[create a new plan](getting-started-creating-a-plan.md).
 
-Each plugin is displayed with a list of plugin-specific, pre-generated plans,
-and an option to [create a new plan](creating-a-plan.md).
+Each plan is displayed with three buttons:
 
-Each plan is displayed with two buttons:
+- A **Split** button that will split the Control Panel in two part and will open the
+plan in the newly created Control Panel. This allow you to see multiple plans at the
+same time, and easily move the Control Panel wherever you want in Atom.
 
 - A **Pin** button, that adds the plan to a list of "pinned" plans displayed above
 the Control Panel.
 
 - A **Play** button that runs/stops the execution of the plan.
 
-![Plugin subpanel](assets/plugin-subpanel.png)
+![Plugin Panel](assets/plugin-panel.png)
 
-Click the Play button next to the npm plan "eslint". Two additional buttons
-should appear under that plan:
+When started, two additional buttons should appear under the plan name.
 
 ![Eslint plan buttons](assets/eslint-plan-buttons.png)
 
-- The Diagnostics button opens the [Diagnostics Subpanel](#diagnostics-subpanel).
+- The Diagnostics button opens the [Diagnostics Panel](#diagnostics-panel).
+
+- The Terminal button opens the [Terminal Panel](#terminal-panel).
 
-- The Terminal button opens the [Terminal Subpanel](#terminal-subpanel).
+#### Watch Mode
 
-As you may have noticed, the Jest plan "Watch mode" runs in, well, watch mode:
```

Comment:
Because it's focused on Jest and may lead the person to think that it's the only tool supporting it, which is not the case.
So I chose to present the principle and then use an exemple.